test/copy.cpp

```diff
@@ -114,16 +114,16 @@ TEST(copy, concat)
     Shape shape{1};
     auto arg0 = make_shared<op::Parameter>(element::f32, shape);
     auto arg1 = make_shared<op::Parameter>(element::f32, shape);
-    NodeVector new_args{make_shared<op::Parameter>(element::f32, shape),
-                        make_shared<op::Parameter>(element::f32, shape)};
+    OutputVector new_args{make_shared<op::Parameter>(element::f32, shape),
+                          make_shared<op::Parameter>(element::f32, shape)};
     size_t axis = 0;
     auto node = make_shared<op::Concat>(NodeVector{arg0, arg1}, axis);
-    auto new_node = node->copy_with_new_args(new_args);
+    auto new_node = node->clone_with_new_inputs(new_args);
     auto node_cast = as_type_ptr<op::Concat>(new_node);
     ASSERT_NE(node_cast, nullptr);
 
     ASSERT_TRUE(nullptr != new_node);
-    ASSERT_TRUE(new_args == new_node->get_arguments());
+    ASSERT_TRUE(new_args == as_output_vector(new_node->get_arguments()));
```

Comment:
Why not `new_args == new_node->input_values()` ?

Reply:
why not put that in a suggestion so I can just click it to accept?